Update documentation - Form Block (#394)
kendraio/kendraio-app: docs/workflow/blocks/mapping.rst

Handy JMESPath patterns
-----------------------

> Using string as value

Reviewer: ```suggestion
Using a string as a value
```

^^^^^^^^^^^^^^^^^^^^

> When manually creating data, a string will be passed as value.

Reviewer: ```suggestion
Data can be made manually using a string value.
```

Author: I rewrote it in a different way, let me know if is understable

> Usually keys and values are wrapped in double quotes, to be valid JMESPath.

Reviewer: This is unnecessarily vague. Usually, or always? If not always, then when? @gsambrotta

Author: mhm I see your point. I rewrite the whole bit. Maybe now is more clear.

In the case of an hardcoded string, the value must be wrapped in backtick, otherwise it will result `null`

.. code-block:: json
  {
    "name": `John`,
    "surname": `Doe`,
    "iAmNull": "not showing"
  }

Output will be:

.. code-block:: text

  name:"John"
  surname:"Doe"
  iAmNull:null

> Is also possible to wrap the whole expression in backtick. The output will be the same:

Reviewer: ```suggestion
We've seen that property values of object keys can be set using backticks, and it is also possible to wrap a whole JSON object in backticks. The output will be the same as this:
```
